### PR TITLE
Ons tabbar test

### PR DESCRIPTION
**Pull request by andipavllo**

### CODE REVIEW

File: core/elements/ons-tabbar.spec.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    element.children[0].classList.add('tab-bar--piyo');
+    element.children[1].classList.add('tab-bar--piyo');
+    element.setAttribute('modifier', 'fuga');
+    expect(element.children[0].classList.contains('tab-bar--piyo')).to.be.true;
+    expect(element.children[1].classList.contains('tab-bar--piyo')).to.be.true;
+    expect(element.children[0].classList.contains('tab-bar--fuga__content')).to.be.true;
+    expect(element.children[1].classList.contains('tab-bar--fuga')).to.be.true;
+  });
+
+  it('accepts \'var\' attribute', function() {
```
Comment (argelius) [inline core/elements/ons-tabbar.spec.js]: This test doesn't seem to do anything. Please remove it.

File: core/elements/ons-tabbar.spec.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+    expect(element.getAttribute('var')).to.equal('myVar');
+  });
+
+  it('has a unique auto generated id', function() {
+    var element = new OnsTabbarElement();
+    expect(element.getTabbarId()).to.be.ok;
+  });
+
+  it('has \'position\' property', function(done) {
```
Comment (argelius) [inline core/elements/ons-tabbar.spec.js]: attribute

**Comment by argelius** [comment]
Can you also test the `active` attribute?

**Comment by andipavllo** [comment]
The active tab test is already there, from row 120. The active attribute is on `ons-tab` element, so I'm implementing the test in `ons-tab.specs.js`

**Comment by argelius** [comment]
Oh, I see. Yeah, maybe it's better to test it in the ons-tab spec.

**Comment by argelius** [comment]
@andipavllo Tabbar also has 'reactive' event

**Comment by argelius** [comment]
Thanks @andipavllo ! I've merged this now.

**Pull request merged**